src/app/examples/list/base-list.component.ts

```diff
@@ -6,90 +6,103 @@ export class BaseListComponent {
       subTitle: '2000 pcs',
       amount: '5.587.218.309 DKK',
       detail: 225,
+      color: 'default',
     },
     {
       id: 1,
       title: 'Cypress Semiconductor Corporation',
       subTitle: '1827 pcs',
       amount: '76.980 DKK',
       detail: -3,
+      color: 'light',
     },
     {
       id: 2,
       title: 'Ultragenyx Pharmaceutical Inc.',
       subTitle: '787 pcs',
       amount: '83.004 DKK',
       detail: -115,
+      color: 'white',
     },
     {
       id: 3,
       title: 'Trans World Entertainment Corp.',
       subTitle: '467 pcs',
       amount: '60.963 DKK',
       detail: 6,
+      color: 'light',
     },
     {
       id: 4,
       title: 'Astronics Corporation',
       subTitle: '791 pcs',
       amount: '33.830 DKK',
       detail: -154,
+      color: 'white',
     },
     {
       id: 5,
       title: 'Riverview Bancorp Inc',
       subTitle: '206 pcs',
       amount: '60.775 DKK',
       detail: 98,
+      color: 'light',
     },
     {
       id: 6,
       title: 'Haemonetics Corporation',
       subTitle: '988 pcs',
       amount: '61.196 DKK',
       detail: 220,
+      color: 'white',
     },
     {
       id: 7,
       title: 'PJT Partners Inc.',
       subTitle: '1706 pcs',
       amount: '52.441 DKK',
       detail: 129,
+      color: 'light',
     },
     {
       id: 8,
       title: 'Total S.A.',
       subTitle: '827 pcs',
       amount: '62.276 DKK',
       detail: 53,
+      color: 'white',
     },
     {
       id: 9,
       title: 'Northrop Grumman Corporation',
       subTitle: '443 pcs',
       amount: '95.190 DKK',
       detail: -135,
+      color: 'white',
     },
     {
       id: 10,
       title: 'Rudolph Technologies, Inc.',
       subTitle: '1799 pcs',
       amount: '18.442 DKK',
       detail: 86,
+      color: 'kirby-color-brightness-white',
```

Comment:
??

Reply:
Just to show this is not being rendered yet

Reply:
Removed